pkg/api/server/doc.go

```diff
@@ -23,44 +23,44 @@
 //
 // See podman-service(1) for more information.
 //
-//  Quick Examples:
+//	Quick Examples:
 //
-//   'podman info'
+//	 'podman info'
 //
-//      curl --unix-socket /run/podman/podman.sock http://d/v4.0.0/libpod/info
+//	    curl --unix-socket /run/podman/podman.sock http://d/v4.0.0/libpod/info
 //
-//   'podman pull quay.io/containers/podman'
+//	 'podman pull quay.io/containers/podman'
 //
-//      curl -XPOST --unix-socket /run/podman/podman.sock -v 'http://d/v4.0.0/images/create?fromImage=quay.io%2Fcontainers%2Fpodman'
+//	    curl -XPOST --unix-socket /run/podman/podman.sock -v 'http://d/v4.0.0/images/create?fromImage=quay.io%2Fcontainers%2Fpodman'
 //
-//   'podman list images'
+//	 'podman list images'
 //
-//      curl --unix-socket /run/podman/podman.sock -v 'http://d/v4.0.0/libpod/images/json' | jq
+//	    curl --unix-socket /run/podman/podman.sock -v 'http://d/v4.0.0/libpod/images/json' | jq
 //
 // Terms Of Service:
 //
 // https://github.com/containers/podman/blob/913caaa9b1de2b63692c9bae15120208194c9eb3/LICENSE
 //
-//     Schemes: http, https
-//     Host: podman.io
-//     BasePath: /
-//     Version: 4.0.0
-//     License: Apache-2.0 https://opensource.org/licenses/Apache-2.0
-//     Contact: Podman <podman@lists.podman.io> https://podman.io/community/
-//
-//     InfoExtensions:
-//     x-logo:
-//       - url: https://raw.githubusercontent.com/containers/libpod/main/logo/podman-logo.png
-//       - altText: "Podman logo"
-//
-//     Produces:
-//     - application/json
-//     - application/octet-stream
-//     - text/plain
-//
-//     Consumes:
-//     - application/json
-//     - application/x-tar
+//	Schemes: http, https
```

Comment:
The generate yaml looks good to me.

Reply:
```
rm -f swagger.yaml
swagger generate spec -o swagger.yaml -i tags.yaml -w ./ -m
yaml: found character that cannot start any token
make[1]: *** [Makefile:13: swagger.yaml] Error 1
make[1]: Leaving directory '/var/tmp/go/src/github.com/containers/podman/pkg/api'
make: *** [Makefile:424: pkg/api/swagger.yaml] Error 2
```

Cannot reproduce on my machine -.-

Reply:
Do we need a newer version of swagger in CI?

Reply:
Yes, it seems like swagger v0.30 comes with Go 1.19 fixes (https://github.com/go-swagger/go-swagger/releases/tag/v0.30.0).

Where do we get swagger from in CI? I fail to find a place where it's being installed? @cevich

Reply:
It is installed into the images at build time, https://github.com/containers/automation_images/blob/f4068fd0ee507d308bbfb34f1f934db19021f6b9/cache_images/podman_tooling.sh#L50-L56

Reply:
Also related https://github.com/containers/podman/issues/15550

Reply:
> It is installed into the images at build time, https://github.com/containers/automation_images/blob/f4068fd0ee507d308bbfb34f1f934db19021f6b9/cache_images/podman_tooling.sh#L50-L56

Thanks!  I strongly believe there should be a make target for installing swagger.  If contributors run into issues, there's absolutely no trace in containers/podman of how to install swagger or which version is being used.

Reply:
@cevich can you help me get images with an updated swagger for this PR?

Reply:
yep, but then the problems need to be fixed too :grin:

Reply:
oh! actually, we already install from a URL with "latest" in the name.  Would it be okay if I did this with my F37 image update or do you need/want it to be separate?

Reply:
That sounds good to me.  But we probably run into chicken-egg thing.  F37 ships Go 1.19 which requires an update golangci-lint which requires the changes in this PR here which require an updated swagger :-D

Reply:
Well we're screwed even more actually.  I can't build VM images at all because **f$#@!&#^** Ubuntu builds are barfing about the kubik podman requiring a (non-existent) netavark package :cursing_face:

So...maybe we just give up on Ubuntu for now...though that implies we're not testing CGv1 through which might be bad
-sigh-

Reply:
https://github.com/containers/automation_images/pull/225